packages/salesforcedx-vscode-core/test/vscode-integration/commands/auth/forceAuthLogout.test.ts

```diff
@@ -191,10 +191,10 @@ describe('Force Auth Logout Default', () => {
     expect(
       notificationStub.callCount,
       'should not have posted an error message'
-    ).to.equal(1);
+    ).to.equal(2);
     const notificationArgs = notificationStub.getCall(0).args;
     expect(notificationArgs).to.deep.equal([
-      'SFDX: Log Out from Default Org successfully ran',
+      'SFDX: Set a Default Org successfully ran',
       'Show',
```

Comment:
I have updated the tests for now and logged a new WI [W-12431853](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001K0MdvYAF/view) to address the two notification pop-ups while running `SFDX: Log Out from Default Org` to log out from a scratch org.